test/new-e2e/system-probe/config/platforms.json

```diff
@@ -193,6 +193,17 @@
         "mantic"
       ]
     },
+    "ubuntu_24.04": {
+      "os_name": "Ubuntu",
+      "os_id": "ubuntu",
+      "kernel": "6.5.0-9-generic",
```

Comment:
My understanding was that ubuntu 24.04 includes kernel 6.8, is this expected ?

Reply:
Yes, I just updated the paths manually so I could test the images, the kernel information is not updated yet. I'll do that once I ensure the image works correctly.